.github/CODEOWNERS

```
@@ -124,6 +124,7 @@ internal/kubelet/                                        @open-telemetry/collect
 internal/metadataproviders/                              @open-telemetry/collector-contrib-approvers @Aneurysm9 @dashpole
 internal/sharedcomponent/                                @open-telemetry/collector-contrib-approvers @open-telemetry/collector-approvers
 internal/splunk/                                         @open-telemetry/collector-contrib-approvers @dmitryax
+internal/sqlquery/                                       @open-telemetry/collector-contrib-approvers @dmitryax @pmcollins
```

Can you add yourself as a code owner?
```suggestion
internal/sqlquery/                                       @open-telemetry/collector-contrib-approvers @crobert-1 @dmitryax
```

---

Done